model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -122,9 +122,9 @@ data class ResolvedLicenseInfo(
         licenseChoices: List<SpdxLicenseChoice>,
         licenseView: LicenseView = LicenseView.ALL
     ): ResolvedLicenseInfo {
-        val licenses = effectiveLicense(licenseView, licenseChoices)?.decompose().orEmpty()
+        val effectiveLicenses = effectiveLicense(licenseView, licenseChoices)?.decompose().orEmpty()
```

Comment:
Unrelated renaming.

Reply:
It's not unrelated: In order to be able to simplify `this.licenses`, the local `licenses` has to be renamed.